amgut/db/patches/0009.sql

```diff
@@ -0,0 +1,29 @@
+-- Add created_on timestamp for handout kits
+ALTER TABLE ag.ag_handout_kits ADD created_on timestamp DEFAULT current_timestamp NOT NULL;
+
+-- Create new handout_barcode table
+CREATE TABLE ag.handout_barcode ( 
+	kit_id               varchar  NOT NULL,
+	barcode              varchar(9)  NOT NULL,
+	sample_barcode_file  varchar(13),
+	CONSTRAINT idx_handout_barcode PRIMARY KEY ( kit_id, barcode )
+ );
+
+CREATE INDEX idx_handout_barcode_0 ON ag.handout_barcode ( barcode );
+
+CREATE INDEX idx_handout_barcode_1 ON ag.handout_barcode ( kit_id );
+
+ALTER TABLE ag.handout_barcode ADD CONSTRAINT fk_handout_barcode FOREIGN KEY ( barcode ) REFERENCES ag.barcode( barcode );
+
+-- Copy needed data to new table
+INSERT INTO ag.handout_barcode (kit_id, barcode, sample_barcode_file)
+SELECT kit_id, barcode, sample_barcode_file FROM ag.ag_handout_kits;
+
+-- Delete duplicate handout rows and unneded columns from handouts table
+DELETE FROM ag.ag_handout_kits WHERE barcode NOT IN (SELECT max(barcode) FROM ag.ag_handout_kits);
```

Comment:
it looks like the barcode isn't necessary then, right? why not just `alter table ag.ag_handout_kits drop column barcode`?